modules/xmpp/ChatRoom.js

```diff
@@ -686,35 +683,48 @@ export default class ChatRoom extends Listenable {
 
     /**
      * Send text message to the other participants in the conference
-     * @param body
+     * @param message
+     * @param elementName
      * @param nickname
      */
-    sendMessage(body, nickname) {
+    sendMessage(message, elementName, nickname) {
         const msg = $msg({ to: this.roomjid,
             type: 'groupchat' });
 
-        msg.c('body', body).up();
+        if (elementName === 'body') {
```

Comment:
Can you please add a short comment which describes the logic? It makes sense, but it might not be obvious for someone reading it 6 months from now :)